src/stpipe/step.py

```diff
@@ -1262,6 +1249,50 @@ def update_pars(self, parameters):
             else:
                 self.log.debug(f'Parameter {parameter} is not valid for step {self}. Ignoring.')
 
+    @classmethod
+    def build_config(cls, input, **kwargs):
+        """Build the ConfigObj to instantiation a Step
+
+        A Step config is built in the following order:
+
+        - CRDS parameter reference file
+        - Local parameter reference file
+        - `Step.call` keyword arguments
```

Comment:
Does this class method only get called from `Step.call`?  Or possibly elsewhere?  Might be good to just to indicate Step keyword args in case we refactor it to be used otherwise?  Does it get called from `Step.from_cmdline`?